docs/cases/cases-overview.asciidoc

```diff
@@ -16,11 +16,13 @@ external systems from within {es-sec}:
 
 <<cases-ui-integrations>> describes how to set up external integrations. When configuring case fields, note that data from mapped fields can be pushed to external systems but cannot be pulled in.
 
-You can create and manage cases via the UI or the <<cases-api-overview>>.
+You can create and manage cases using the UI or the <<cases-api-overview>>.
 
-NOTE: To send cases to external systems, you need the
+IMPORTANT: To send cases to external systems, you need the
 https://www.elastic.co/subscriptions[appropriate license] and your role needs *All* privileges for the *Action and Connectors* feature. For information about the base role privileges you need to access cases, refer to <<case-permissions>>.
 
+NOTE: From {es-sec}, you cannot access cases created in {observability} or Stack Management.
```

Comment:
`elastic-sec` is the [formal attribute](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L235) -- just want to make sure we don't have rendering issues. 
```suggestion
NOTE: From {elastic-sec}, you cannot access cases created in {observability} or Stack Management.
```